test/integration/auth.spec.ts

```diff
@@ -890,6 +890,7 @@ describe('admin.auth', () => {
           return admin.auth().verifyIdToken(currentIdToken, true);
         })
         .then((decodedIdToken) => {
+          expect(decodedIdToken.sub).to.equal(uid);
```

Comment:
I am not sure how this solves the issue I pointed out earlier.

How about you use async/await instead of chained promises in these tests. It would be cleaner to read too.
```
....
// If this throws, it won't be caught and the test will fail.
const decodedIdToken = await admin.auth().verifyIdToken(currentIdToken, false);
expect(decodedIdToken.uid).to.equal(uid);
try {
  await admin.auth().verifyIdToken(currentIdToken, true);
} catch (e) {
  // You know for sure that this was thrown by the second call.
  expect(error).to.have.property('code', 'auth/user-disabled');
}
```